tests/wpt/tests/css/CSS2/floats/floats-wrap-bfc-with-margin-010.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<link rel="author" title="Oriol Brufau" href="obrufau@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css2/#floats">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
```

Comment:
Might be nice to put a little description here of what this testing.

Reply:
Done. I have also modified the test to check more things.